rockfield/style.css

```diff
@@ -2869,7 +2869,7 @@ body:not(.fse-enabled) .site-description {
 	z-index: 1;
 }
 
-.main-navigation > div > ul li:hover, .main-navigation.js-focus-within > div > ul li[focus-within], .js-focus-within .main-navigation > div > ul li[focus-within] {
+.main-navigation > div > ul li:hover, .main-navigation > div > ul li[focus-within] {
```

Comment:
This is undoing changes made in https://github.com/Automattic/themes/pull/8469 by mistake